etc/openqa/openqa.ini

```diff
@@ -149,6 +149,13 @@ blocklist = job_grab job_done
 #exchange = pubsub
 #topic_prefix = suse
 
+# Cleanup related configuration (besides limits)
+[cleanup]
+# Specifies whether different cleanup tasks can run in parallel. This option
+# makes particularly sense if assets and results are stored on different storage
+# locations or if your common storage solution is performant enough.
+concurrent = 0
```

Comment:
For some reason I thought this might take a number of tasks to run in parallel. Looking at the implementation clarified that of course... might be clear enough anyway, just wanted to mention it

Reply:
I hope the word "whether" in the comment makes it clear that this is just a flag.

Reply:
would it be possible to use bools like "true" or "false" here? This would clarify all ambiguity. If not I think mentioning it in the comment would make sense

Reply:
In Perl there's no true/false do this would require extra handling and would be inconsistent with other settings.